config/app_local.example.php

```diff
@@ -74,6 +74,15 @@
         ],
     ],
 
+    /*
+     * Debugger configuration
+     *
+     * See app.php for more info.
```

Comment:
It should be removed from app.php. Having a config at 2 places will just confuses new users.

Reply:
but I prefer to kept it in app.php , because it saves in VCS , and it avoids to forgetting config name

also some of other configs are in 2 places , and no problem
anyway I can comment original code , but removing is not good idea IMO

Reply:
If you are going to keep it in app.php then what's the point in having it in the local config file?

Anyway you are free to do do whatever you want in your app but having the duplication is not a good default for the app skeleton.

Reply:
@saeideng I think it is pretty clear that every value in app.php can be overwritten in local file
In fact, this will be the case for also many other options as per local dev.
So I guess keeping it in the main file suffices for most - as they know how to copy paste when it becomes necessary.

With this we keep the local file concise and minimal, makes it also easier to review.

Reply:
> what's the point in having it in the local config file?


2 team members could have 2 different editor